crates/extensions/c8y_mapper_ext/src/converter.rs

```diff
@@ -1030,9 +1030,8 @@ impl CumulocityConverter {
                 self.process_alarm_messages(&source, message, alarm_type)
             }
 
-            Channel::Command { cmd_id, .. } if message.payload_bytes().is_empty() => {
-                // The command has been fully processed
-                self.active_commands.remove(cmd_id);
+            Channel::CommandMetadata { .. } if message.payload_bytes().is_empty() => {
+                // Clearing command capabilities
```

Comment:
Is this enough? Shouldn't we remove the corresponding operation file and send the updated operations list?